### ISSUE TITLE
Structure from other Mods not generating

### ISSUE THREAD

**Issue by SlayDigger** [CorgiTaco-Archive/overworld-two-forge#1]
Right now, structures from other mods do not generate in the Overworld Two world type. I tried Valhelsia Structures, The Farlanders, and Towers of The Wild, all of which didn't generate any structures. They work in other things like Default, Biomes O Plenty, and Terraforged, so it isn't on their side from my knowledge.

**Comment by SlayDigger** [comment]
Hold on, some other people are testing and getting structures

**Comment by Teacyn** [comment]
I tested this myself and I was able to get structures from Valhesia, Towers of the Wild, Repurposed Structures, and YUNGs' Better Mineshafts. 

However, I generated 20 using OW2 and none of them allowed me to find any structures from Dungeons Plus or Conjurer, both of which use Structure Gel API. As such, I believe there may be an incompatibility there. I reported this as an issue over on that GitLab, in case it needs to be addressed from their end. https://gitlab.com/modding-legacy/structure-gel-api/-/issues/12
